cli/cli/commands/route.py

```diff
@@ -0,0 +1,65 @@
+import click
+
+from cli import client
+from cli.gateway import GatewayManager
+from cli.infra import InfraManager
+from cli.model import Route
+
+
+@click.group()
+def route():
+    """Manage gateway routes"""
+    pass
+
+
+@route.command()
+def ls():
+    """Print the routes configured on the gateway"""
+    manager = GatewayManager()
+    manager.print_routes()
+
+
+@route.command()
+@click.argument("relative_url")
+@click.argument("target")
```

Comment:
This is silly and completely unrelated, but it would be cool to offer autocompletion based on the URLs of already deployed functions for the target parameter. I think it's possible with click: https://click.palletsprojects.com/en/8.1.x/shell-completion/#overriding-value-completion